diff --git a/node_modules/intl/index.js b/node_modules/intl/index.js
index 599cf0b..f25643d 100644
--- a/node_modules/intl/index.js
+++ b/node_modules/intl/index.js
@@ -1,6 +1,9 @@
 // Expose `IntlPolyfill` as global to add locale data into runtime later on.
+
+var global={};
 global.IntlPolyfill = require('./lib/core.js');
 
+
 // Require all locale data for `Intl`. This module will be
 // ignored when bundling for the browser with Browserify/Webpack.
 require('./locale-data/complete.js');
